### PR TITLE
Upkeeping: Links, language, style

### PR DESCRIPTION
**Pull request by Sebastian-Hojas**
Fixes a small number of things:
- Broken links and TOC
- Adds Alamofire next to AFNetworking
- Minor language, formatting and references mistakes
- Removes section on deprecated XCode Instrument's Automation tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,9 +352,10 @@ You can include the original [vector graphics (PDFs)][vector-assets] produced by
 
 ### Naming
 
-Apple pays great attention to keeping naming consistent, if sometimes a bit verbose, throughout their APIs. When developing for Cocoa, you make it much easier for new people to join the project if you follow [Apple's naming conventions][cocoa-coding-guidelines].
+Apple pays great attention to keeping naming consistent. Adhering to their [coding guidlines for Objective-C][cocoa-coding-guidelines] and [API design guidlines for Swift][swift-api-design-guidlines] makes it much easier for new people to join the project.
```
Comment (erikjalevik) [inline README.md]: Fix typos: guidlines - > guidelines

File: README.md [hunk 2]
```diff
@@ -201,20 +201,21 @@ Generally speaking, make it a conscious decision to add an external dependency t
...
 
 [timezones-youtube]: https://www.youtube.com/watch?v=-5wpm-gesOY
 [datetools-github]: https://github.com/MatthewYork/DateTools
 
 ### Auto Layout Libraries
-If you prefer to write your views in code, chances are you've met either of Apple's awkward syntaxes – the regular `NSLayoutConstraint` factory or the so-called [Visual Format Language][visual-format-language]. The former is extremely verbose and the latter based on strings, which effectively prevents compile-time checking. Fortunately, they've addressed the issue in iOS 9, allowing [a more concise specification of constraints][nslayoutanchor].
+If you prefer to write your views in code, chances are you've heard of either Apple's awkward syntaxes – the regular `NSLayoutConstraint` factory or the so-called [Visual Format Language][visual-format-language]. The former is extremely verbose and the latter based on strings, which effectively prevents compile-time checking. Fortunately, they've addressed the issue in iOS 9, allowing [a more concise specification of constraints][nslayoutanchor].
```
Comment (erikjalevik) [inline README.md]: Stylistic change.

File: README.md [hunk 3]
```diff
@@ -110,13 +110,13 @@ will update all pods to the newest versions permitted by the Podfile. You can us
 
 [Carthage][carthage] takes the ["simple, not easy"][simple-made-easy] approach by building your dependencies into binary frameworks, without magically integrating them with your project in any way. This also greatly reduces build times, because your dependencies have already been compiled by the time you start building.
 
-There is no centralized repository of projects, which means any library that can be compiled into a framework supports Carthage out of the box. Note that dynamic frameworks are only available from iOS 8 onwards.
+There is no centralized repository of projects, which means any library that can be compiled into a framework supports Carthage out of the box. Note that dynamic frameworks are only available from iOS 8 onward.
```
Comment (erikjalevik) [inline README.md]: Stylistic change. Although personally, I think I prefer onwards.

http://english.stackexchange.com/questions/59137/onward-vs-onwards

File: README.md (hunk 2 above)
Comment (Sebastian-Hojas) [inline README.md]: I'm definitely no native, but I don't think you can say "to meet _either of_ A or B". I think you can only say "to meet _either_ A or B" or "to _hear of_ either A or B".

Comment (erikjalevik) [inline README.md]: It sounds perfectly natural to my ears (also see plenty of examples at http://www.linguee.fr/anglais-francais/traduction/meet+either+of+the+above.html), but your version is arguably less convoluted, so let's go with it. :)

Comment (erwald) [inline README.md]: To me, the new wording sounds wrong – I would write "chances are you've come across one of Apple's awkward syntaxes ..."

Comment (erikjalevik) [inline README.md]: Oh shit, you're right, there's a missing "of". Seems my brain just auto-filled it.

Comment (erikjalevik) [inline README.md]: Although I'm perfectly happy with the original wording as it is.

Comment (erwald) [inline README.md]: I agree – either original wording, or new wording with "of" added (or my suggestion) would be fine.

**Pull request merged**